autodoc_traits/__init__.py

```diff
@@ -1,7 +1,2 @@
-__version__ = "1.0.0"
-
-
-def setup(app, *args, **kwargs):
-    from .autodoc_traits import setup
-
-    return setup(app, *args, **kwargs)
+from ._version import __version__  # noqa
+from .autodoc_traits import ConfigurableDocumenter, TraitDocumenter, setup  # noqa
```

Comment:
I think there's actually no reason to have more than just one autodoc_traits.py (no directory at all) for a single-purpose package like this, what do you think?